libs/nucs/src/token.rs

```diff
@@ -43,13 +43,17 @@ pub struct NucToken {
     #[serde(default)]
     pub meta: Option<JsonObject>,
 
+    /// The token nonce.
+    #[serde(serialize_with = "hex::serde::serialize", deserialize_with = "hex::serde::deserialize")]
+    pub nonce: Vec<u8>,
```

Comment:
We should probably enforce the max length of the `nonce` somewhere?

Reply:
I think the enforcement should come at the entire token size, not just here. e.g. how is a very long nonce different from, say, a very long `args` parameter? All of these checks will be applied later once we have the models that sit on top of this one.

Reply:
good point!